recipes/centos_spring_boot/Dockerfile

```diff
@@ -14,9 +14,8 @@ ARG SPRING_BOOT_VERSION=1.4.1.RELEASE
 ARG JUNIT_VERSION=4.12
 
 ENV SPRING_BOOT_GROUP=org.springframework.boot
+ADD configure_maven_and_deps.sh /home/user/
 
-COPY install_spring_boot_dependencies.sh /tmp/
-RUN sudo chown user:user /tmp/install_spring_boot_dependencies.sh && \
-    sudo chmod a+x /tmp/install_spring_boot_dependencies.sh && \
-    scl enable rh-maven33 /tmp/install_spring_boot_dependencies.sh && \
-    sudo rm -f /tmp/install_spring_boot_dependencies.sh
+RUN sudo chmod +x ${HOME}/configure_maven_and_deps.sh
+
+CMD ["sh","-c","${HOME}/configure_maven_and_deps.sh"]
```

Comment:
why is it a replaced by a command.
it was a run instruction so it was to cache maven dependencies during the build of the image, not to do it when the container is started ?

Reply:
since we have settings.xml  in place and the `MAVEN_MIRROR_URL` is configurable environment variable, we  need to have entrypoint kind of scripting which can set the MAVEN_MIRROR url as configured by the user.  Also as mentioned earlier comment  I don't see a big value with installed deps during build of images,  as anyways running your build inside spring boot, vert.x or other maven based app in side che will pull it automatically

Reply:
it's to build the examples faster. Downloading dependencies at runtime are making user experience not so good.
This should be not removed as this enhancement was requested.

Reply:
but @benoitf - for the stacks in samples - we are not downloading all - it just downloads a very tiny subset, still even today if you see there is bunch of other dependencies that are downloaded e.g. I could have a project where `spring-boot-loader-tools` might not be needed at all .. then why should we download it ?

Reply:
I think you should separate your commits of the PR.
Only add support of MAVEN_MIRROR_URL, not cleaning up what is already done

Reply:
OK, if you wish so I dont mind ! will send a updated PR

Reply:
@benoitf opened new PR #143, i shall have this closed right now keeping it for comments refernece